.circleci/setupAndTestWithFreeCore.sh

```diff
@@ -55,3 +55,25 @@ if ! [[ -z "${CIRCLE_NODE_TOTAL}" ]]; then
 else
     TEST_MODE=testing SUPERTOKENS_CORE_TAG=$coreTag NODE_PORT=8081 INSTALL_PATH=../supertokens-root npm test
 fi
+
+API_PORT=3030
+ST_CONNECTION_URI=http://localhost:8081
+
+# start test-server
+pushd test/test-server
+npm install
+API_PORT=$API_PORT ST_CONNECTION_URI=$ST_CONNECTION_URI npm start &
+popd
+
+# run tests
+cd ../
+git clone git@github.com:supertokens/backend-sdk-testing.git
+cd backend-sdk-testing
+git checkout 641af3fbe8d6b7a064726be82244df3ff4b32bce # checkout latest
```

Comment:
Will the FDI based test checkout be done in a separate PR?

Reply:
yes